### PR TITLE
Adding max_tokens & i/o -token pricing

### LINKED ISSUE #363: [Bug]: Model Cost dictionary missing remaining cohere, anthropic, etc. models
### What happened?

The model cost dictionary - https://github.com/BerriAI/litellm/blob/main/model_prices_and_context_window.json

does not cover all the default litellm models (e.g. cohere) https://github.com/BerriAI/litellm/blob/1913d36e053cc1d6cd8d26b70d6a175d88a1f42c/litellm/__init__.py#L95

This needs to be updated to reflect that 

### Relevant log output

_No response_

### Twitter / LinkedIn details

_No response_

### PR DESCRIPTION
**Pull request by WilliamEspegren**
In progress

Fixes #363

### CODE REVIEW

**Comment by ghost** [comment]
@WilliamEspegren we already do this - we make an API call to this, as it's easier to update and ensure everyone has the latest prices + model list. 

https://github.com/BerriAI/litellm/blob/main/model_prices_and_context_window.json

**Comment by WilliamEspegren** [comment]
@krrishdholakia yeah, just added some models and their max_tokens, price per input and price per output token 👍

**Comment by ishaan-jaff** [comment]
@WilliamEspegren fyi looks like this PR deletes some new models we have in __init__.py 

can you ensure the changes in this PR don't remove any of our existing models

**Comment by ghost** [comment]
> @krrishdholakia yeah, just added some models and their max_tokens, price per input and price per output token 👍

why not add it here: https://github.com/BerriAI/litellm/blob/main/model_prices_and_context_window.json

since the package will get those anyway?

File: model_prices_and_context_window.json [hunk 1]
```diff
@@ -39,61 +54,81 @@
         "input_cost_per_token": 0.000003,
         "output_cost_per_token": 0.000004
     },
-    "gpt-3.5-turbo-instruct": {
-        "max_tokens": 8192,
-        "input_cost_per_token": 0.0000015,
+    "text-ada-001": {
```
Comment (WilliamEspegren) [inline model_prices_and_context_window.json:62]: @krrishdholakia I have added the models to https://github.com/BerriAI/litellm/blob/main/model_prices_and_context_window.json. Check it out under "Files changed"  -> model_prices_and_context_window.json

Comment (ghost) [reply model_prices_and_context_window.json:62]: why is gpt-3.5-turbo-instruct removed? @WilliamEspegren 

can you revert any changes to `__init__.py`

Comment (WilliamEspegren) [reply model_prices_and_context_window.json:62]: The gpt-3.5-turbo-intruct model is included check the file. I can revert changes made to __ini__.py (mostly just autopep8 formatting)

Comment (WilliamEspegren) [reply model_prices_and_context_window.json:62]: @krrishdholakia The changes to __init__.py are reverted

**Comment by WilliamEspegren** [comment]
@krrishdholakia Ready for merge 👍. Is there anything else I should add?

File: litellm/__init__.py [hunk 2]
```diff
@@ -331,4 +331,4 @@ def identify(event_details):
     BudgetExceededError
 
 )
-from .budget_manager import BudgetManager
+from .budget_manager import BudgetManager
```
Comment (ishaan-jaff) [inline litellm/__init__.py:334]: can we remove this change

File: model_prices_and_context_window.json [hunk 3]
```diff
@@ -14,17 +19,27 @@
...
+        "input_cost_per_token": 0.00006,
+        "output_cost_per_token": 0.00012
+    },
     "gpt-3.5-turbo": {
         "max_tokens": 4097,
         "input_cost_per_token": 0.0000015,
         "output_cost_per_token": 0.000002
     },
-    "gpt-3.5-turbo-0613": {
+    "gpt-3.5-turbo-0301": {
```
Comment (ishaan-jaff) [inline model_prices_and_context_window.json:37]: we don't need this change, looks like it's just flipping the order

File: model_prices_and_context_window.json [hunk 4]
```diff
@@ -44,56 +59,81 @@
         "input_cost_per_token": 0.0000015,
         "output_cost_per_token": 0.000002
     },
-    "claude-instant-1": {
```
Comment (ishaan-jaff) [inline model_prices_and_context_window.json:47]: can we not delete claude

File: model_prices_and_context_window.json [hunk 5]
```diff
@@ -44,56 +59,81 @@
...
-    "claude-2": {
-        "max_tokens": 100000,
-        "input_cost_per_token": 0.00001102,
-        "output_cost_per_token": 0.00003268
+    "text-ada-001": {
+        "max_tokens": 2049,
+        "input_cost_per_token": 0.0000004,
+        "output_cost_per_token": 0.0000004
     },
-    "text-bison-001": {
```
Comment (ishaan-jaff) [inline model_prices_and_context_window.json:62]: can we not delete models

File: model_prices_and_context_window.json [hunk 6]
```diff
@@ -44,56 +59,81 @@
...
+        "input_cost_per_token": 0.00002,
+        "output_cost_per_token": 0.00002
+    },
     "command-nightly": {
         "max_tokens": 4096,
         "input_cost_per_token": 0.000015,
         "output_cost_per_token": 0.000015
     },
-     "command": {
+    "command": {
```
Comment (ishaan-jaff) [inline model_prices_and_context_window.json]: can we fix this formatting change - it's not needed

**Review by ishaan-jaff — CHANGES_REQUESTED** [review]

File: litellm/__init__.py (hunk 2 above)
Comment (WilliamEspegren) [reply litellm/__init__.py:334]: It is my git client that removes new line characters at the bottom of the file, does not change the functionality of the program.

File: model_prices_and_context_window.json (hunk 3 above)
Comment (WilliamEspegren) [reply model_prices_and_context_window.json:37]: @ishaan-jaff Flipping the order to follow the order they are defined in __init__.py. This change makes it easier to confirm whether or not the model/models defined in [__init__.py](https://github.com/BerriAI/litellm/blob/main/litellm/__init__.py) are included in [model_prices_and_context_window.json](https://github.com/BerriAI/litellm/blob/main/model_prices_and_context_window.json).

File: litellm/__init__.py (hunk 2 above)
Comment (WilliamEspegren) [reply litellm/__init__.py:334]: @ishaan-jaff My git client removes new line characters "\n" at the enf of each file. Does not effect the functionality of the program.

File: model_prices_and_context_window.json (hunk 4 above)
Comment (WilliamEspegren) [reply model_prices_and_context_window.json:47]: @ishaan-jaff Look further down the file. Every claude model is included and in the order and place where it is defined in [__init__.py](https://github.com/BerriAI/litellm/blob/main/litellm/__init__.py). This change makes it easier to confirm whether or not the model/models defined in [init.py](https://github.com/BerriAI/litellm/blob/main/litellm/__init__.py) are included in [model_prices_and_context_window.json](https://github.com/BerriAI/litellm/blob/main/model_prices_and_context_window.json).

File: model_prices_and_context_window.json (hunk 5 above)
Comment (WilliamEspegren) [reply model_prices_and_context_window.json:62]: @ishaan-jaff Look further down the file. They are included, just moved to align with the where they are defined in `__init__.py`

**Comment by ishaan-jaff** [comment]
@WilliamEspegren are you still working on this PR ?

**Pull request closed without merging**